platform/definition.go

```diff
@@ -28,13 +28,23 @@ const (
 	JuniperJunos = "juniper_junos"
 	// NokiaSrl is a constant representing the platform string/name for Nokia SRL/SRLinux devices.
 	NokiaSrl = "nokia_srl"
+	// ArubaOs is a constant representing the platform string/name for Aruba OS devices.
```

Comment:
lets update the values here according to whatever we update in the platform names